postman-collection/IoT Central.postman_collection.json

```diff
@@ -2483,6 +2521,345 @@
 					"response": []
 				}
 			]
+		},
+		{
+			"name": "File Upload Storage Configuration",
+			"item": [
+				{
+					"name": "Create a storage account",
+					"request": {
+						"method": "PUT",
+						"header": [
+							{
+								"warning": "This is a duplicate header and will be overridden by the Authorization header generated by Postman.",
```

Comment:
This doesn't look right. You need to change the authorization in the UI from "Inherit auth from parent" to "Bearer Token" on the Authorization tab.